src/main/java/nl/ordina/jobcrawler/scrapers/JobBirdScraper.java

```diff
@@ -0,0 +1,231 @@
+package nl.ordina.jobcrawler.scrapers;
+
+import lombok.extern.slf4j.Slf4j;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.model.VacancyURLs;
+import nl.ordina.jobcrawler.service.ConnectionDocumentService;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+MylerVacancyScraper.java takes care of scraping vacancies from myler.nl
+At this moment getVacancies() returns a list with Vacancies that only exist of a vacancyURL and the broker. No details available (yet).
+ */
+
+@Slf4j
+@Component
+public class JobBirdScraper extends VacancyScraper {
+
+    //private static final String SEARCH_URL = "https://www.jobbird.com/nl/vacature?s=java&rad=30&ot=date";
+    private static final String SEARCH_URL = "https://www.jobbird.com/nl/vacature?s=java";
+    private static final String BROKER = "Jobbird";
+
+    public JobBirdScraper(ConnectionDocumentService connectionDocumentService) {
+        super(connectionDocumentService, SEARCH_URL, BROKER);
+    }
+
+
+    private String createSearchURL(int aPagenr) throws Exception {
+        if (aPagenr == 1) {
+            return SEARCH_URL + "&rad=30&ot=date";
+        } else
+        if (aPagenr > 1) {
+            return SEARCH_URL + "&rad=30&page=" + aPagenr + "&ot=date";
+        } else {
+            throw new Exception("JobBirdScraper:createSearchURL: pagenr must be 1 or greater");
+        }
+    }
+
+    @Override
+    protected List<VacancyURLs> getVacancyURLs() throws IOException {
+        //  Returns a List with VacancyURLs
+        ArrayList<VacancyURLs> URLs = new ArrayList<>();
+
+        try {
+            // on each page including this first one are vacancies to be found
+            Document doc = getDocument(createSearchURL(1));
+            int totalNrOfPages = getTotalNumberOfPages(doc);
+            for (int i=1; i<=totalNrOfPages; i++) {
+                if (i>1)
+                    doc = getDocument(createSearchURL(i));
```

Comment:
As a matter of style I'd prefer to always use code blocks after an if statement. This reduces the chance for bugs later in development. By omitting the curly braces you're increasing the cognitive load of the next programmer, because that person needs to notice that he's not working in a code block.

Reply:
adapted